javascript_nodejs/1.console-echobot/index.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const botbuilder = require('botbuilder');
+const path = require('path');
+const config = require('dotenv').config;
+
+const mainDialog = require('./dialogs/mainDialog');
+
+// load environment variables from .env file.
+const ENV_FILE = path.join(__dirname, '..', '.env');
```

Comment:
Your .env file would typically be at root so unclear if we need '..' here. Would be better with path.join(__dirname, '.env');